controllers/jenkins/pipelinerun/pipelinerun_controller.go

```diff
@@ -161,25 +161,52 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		return ctrl.Result{}, err
 	}
 
-	// requeue immediately
+	// requeue after 1 second
 	return ctrl.Result{RequeueAfter: 1 * time.Second}, nil
 }
 
-func (r *Reconciler) triggerJenkinsJob(devopsProjectName, pipelineName string, prSpec *devopsv1alpha4.PipelineRunSpec) (*job.PipelineBuild, error) {
-	boClient := job.BlueOceanClient{JenkinsCore: r.JenkinsCore, Organization: "jenkins"}
-	return boClient.Build(job.BuildOption{
-		Pipelines:  []string{devopsProjectName, pipelineName},
+func (r *Reconciler) triggerJenkinsJob(projectName, pipelineName string, prSpec *devopsv1alpha4.PipelineRunSpec) (*job.PipelineBuild, error) {
+	c := job.BlueOceanClient{JenkinsCore: r.JenkinsCore, Organization: "jenkins"}
```

Comment:
The parameter `Organization` almost is `jenkins`. Please consider make it be a default value in the Jenkins client library.

Reply:
I've created [an issue](https://github.com/jenkins-zh/jenkins-client/issues/17) to track your suggestion. And it will be completed in another PR.